c10/util/Exception.cpp

```diff
@@ -14,8 +15,7 @@ Error::Error(
     const std::string& backtrace,
     const void* caller)
     : msg_stack_{new_msg}, backtrace_(backtrace), caller_(caller) {
```

Comment:
nit: non-braced initialization?

Reply:
I think the braced initialization here is intentional, they want to create a 1-element vector containing just new_msg

Reply:
I mean, why not backtrace_{backtrace} and caller_{caller}?

Reply:
There's like a chapter in the Modern C++ book about this but I don't remember what it says XD